lib/fluent/plugin/out_bigquery.rb

```diff
@@ -143,6 +143,10 @@ def configure_for_load(conf)
     config_param :request_timeout_sec, :time, default: nil
     config_param :request_open_timeout_sec, :time, default: 60
 
+    ## Partitioning
+    config_param :time_partitioning_type, :string, default: nil
```

Comment:
I think that `:enum` type is better than `:string`.
Because acceptable time partitioning type is specific value. (probably in future too).
`:enum` type can guard invalid parameter on boot timing.